lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -66,16 +66,18 @@ def define
         end
 
         next_node do
-          if calculator.ceremony_country == 'monaco'
-            if calculator.want_to_get_married?
+          if calculator.want_to_get_married?
+            if calculator.ceremony_country == 'monaco'
               outcome :outcome_marriage_in_monaco
             else
```

Comment:
Out of interest, how do we know that the `else` here means the outcome is one in France or french overseas territory? It seems as if some logic before this must have restricted the possibilities so this `else` works. If so that means the logic here is coupled to that other logic which seems like a bad thing. Obviously this is out-of-scope for this PR, but I thought I'd mention it.